src/test/java/io/jenkins/plugins/analysis/core/columns/IssuesTotalColumnTest.java

```diff
@@ -153,8 +155,14 @@ private void verifySumOfChecksStyleAndSpotBugs(final IssuesTotalColumn column) {
         assertThat(column.getTotal(job)).hasValue(1 + 2);
         assertThat(column.getUrl(job)).isEmpty();
 
+        for (ResultAction resultAction : job.getLastCompletedBuild().getActions(ResultAction.class)) {
+            when(resultAction.getOwner()).thenReturn(mock(Run.class));
```

Comment:
I wonder that this works, typically you need to split `when` and `mock`.
```suggestion
            Run<?> run = mock(Run.class);
            for (ResultAction resultAction : job.getLastCompletedBuild().getActions(ResultAction.class)) {
                when(resultAction.getOwner()).thenReturn(run);
            }
```

Reply:
It seems simply using the `when().thenReturn()` will cause compiler errors due to the wildcards. I can't find another way to solve this except:
```
Run<?, ?> build = mock(Run.class);
doReturn(build).when(resultAction).getOwner();
````

Also, I moved it into `createAction()` in `JobStubs`, because many test cases are using `getUrl()`. In this way, I can reduce the duplicate code.